packages/flutter/test/rendering/wrap_test.dart

```diff
@@ -12,17 +12,152 @@ void main() {
     expect(
       renderWrap.toStringDeep(minLevel: DiagnosticLevel.info),
       equalsIgnoringHashCodes(
-        'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
-        '   parentData: MISSING\n'
-        '   constraints: MISSING\n'
-        '   size: MISSING\n'
-        '   direction: horizontal\n'
-        '   alignment: start\n'
-        '   spacing: 0.0\n'
-        '   runAlignment: start\n'
-        '   runSpacing: 0.0\n'
-        '   crossAxisAlignment: 0.0\n'
+          'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
+          '   parentData: MISSING\n'
+          '   constraints: MISSING\n'
+          '   size: MISSING\n'
+          '   direction: horizontal\n'
+          '   alignment: start\n'
+          '   spacing: 0.0\n'
+          '   runAlignment: start\n'
+          '   runSpacing: 0.0\n'
+          '   crossAxisAlignment: 0.0\n'),
+    );
+  });
+
+  test('Compute intrinsic height test', () {
+    final List<RenderBox> children = <RenderBox>[
+      RenderConstrainedBox(
+        additionalConstraints: const BoxConstraints(
+          minWidth: 80,
+          minHeight: 80,
+        ),
+      ),
+      RenderConstrainedBox(
+        additionalConstraints: const BoxConstraints(
+          minWidth: 80,
+          minHeight: 80,
+        ),
+      ),
+      RenderConstrainedBox(
+        additionalConstraints: const BoxConstraints(
+          minWidth: 80,
+          minHeight: 80,
+        ),
       ),
+    ];
+
+    final RenderWrap renderWrap = RenderWrap();
+
+    // ignore: avoid_function_literals_in_foreach_calls
+    children.forEach(
+          (RenderBox child) {
+        renderWrap.add(child);
+      },
+    );
```

Comment:
here and below: remove the ignore and just write:

```suggestion
children.forEach(renderWrap.add);
```